lib/presentation/ui/about/contacts.dart

```diff
@@ -0,0 +1,11 @@
+enum ContactType{
+  PHONE, URL, EMAIL
+}
+
+class Contact{
```

Comment:
I'm assuming this is about the ContactType?

Reply:
What will the parameter 'info' contain? Will it be used for the email address, URL or phone number? I thought of making a constructor like: 
```dart
Contact.email(this.name, this.imageUrl, this.info, {this.type = ContactType.email});
``` 
If the info is used as mentioned above this will work.

Reply:
Info can be a String for a description

The type being an optional parameter is great 👍